distribution/tools/plugin-cli/src/main/java/org/opensearch/plugins/InstallPluginCommand.java

```diff
@@ -218,7 +218,7 @@
             Arrays.asList("b", "batch"),
             "Enable batch mode explicitly, automatic confirmation of security permission"
         );
-        this.arguments = parser.nonOptions("plugin id");
+        this.arguments = parser.nonOptions("plugin <name|URL>");
```

Comment:
This is nice, we do also support installing by file. Could add that as an option as well?
For example: https://github.com/opensearch-project/opensearch-plugins/blob/main/INSTALLING.md#installing-opensearch-plugins

Reply:
I've added more descriptions to the help command, as seen below. 
```
Install a plugin

Plugins are packaged as zip files. Each packaged plugin must contain a plugin properties file.

The install command takes a plugin id, which may be any of the following:
  An official opensearch plugin name
  Maven coordinates to a plugin zip
  A URL to a plugin zip

The following official plugins may be installed by name:
  analysis-icu
  analysis-kuromoji
  analysis-nori
  analysis-phonetic
  analysis-smartcn
  analysis-stempel
  analysis-ukrainian
  discovery-azure-classic
  discovery-ec2
  discovery-gce
  ingest-attachment
  mapper-annotated-text
  mapper-murmur3
  mapper-size
  repository-azure
  repository-gcs
  repository-hdfs
  repository-s3
  store-smb
  transport-nio

Non-option arguments:                 
[String] -- plugin <name|Zip File|URL>

Option             Description                                            
------             -----------                                            
-E <KeyValuePair>  Configure a setting                                    
-b, --batch        Enable batch mode explicitly, automatic confirmation of
                     security permission                                  
-h, --help         Show help                                              
-s, --silent       Show minimal output                                    
-v, --verbose      Show verbose output   
```